packages/server-core/src/createApp.ts

```diff
@@ -185,6 +186,12 @@ export const createFeathersExpressApp = (
     return res.status(204).send()
   })
 
+  app.post('/api/credentials/request', IssueRequest.post)
```

Comment:
lets use a feathers route instead of a raw REST endpoint

Reply:
Don't know how to do that, tho happy to refactor in the next PR.